src/rdymesh.c

```diff
@@ -491,9 +493,12 @@ static PetscErrorCode RDyEdgesCreateFromDM(DM dm, RDyEdges *edges) {
   }
 
   // fetch global edge IDs.
-  ISLocalToGlobalMapping map;
-  PetscCall(DMGetLocalToGlobalMapping(dm, &map));
```

Comment:
So we don't need these `*LocalToGlobal*` calls in the parallel case? Sorry--haven't looked at this stuff in a while. :-)

Reply:
This `*LocalToGlobal*` is only correct cells and not valid for edges because our DOFs are on cells. We need some new code (either in RDycore or in PETSc) to correctly set global edge IDs for parallel runs, which will be fixed in a future PR.